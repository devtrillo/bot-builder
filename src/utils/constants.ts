export const PRODUCT_NAME = "Bot builder";

if (import.meta.vitest) {
  const { it, expect } = import.meta.vitest;
  it("should not change any of the variables", () => {
    expect(PRODUCT_NAME).toMatchSnapshot();
  });
}
