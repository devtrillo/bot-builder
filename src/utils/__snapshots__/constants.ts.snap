// Vitest Snapshot v1

exports[`add 1`] = `"Bot builder"`;

exports[`should not change any of the variables 1`] = `"Bot builder"`;
